fs/walk.ts

```diff
@@ -95,7 +95,14 @@ function include(f: FileInfo, options: WalkOptions): Boolean {
   if (options.exts && !options.exts.some(ext => f.path.endsWith(ext))) {
     return false;
   }
-  if (options.match && !options.match.some(pattern => pattern.test(f.path))) {
+  if (
+    options.match &&
+    !options.match.some(pattern => {
```

Comment:
probably need this for skip too. _may_ be worth putting into function:

some(patterns, path) = ...

should also have a comment, and perhaps link to SO, as this is a very surprising thing to have to do.